tests/Aspire.Components.Common.Tests/Aspire.Components.Common.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsTestSupportProject>true</IsTestSupportProject>
```

Comment:
NIT: in the tests/testproject seems like you defined this in Directory.Build.props (probably due to some race condition of wantint to set before parent's D.B.p). Don't you need the same here?

Reply:
Yep, it should be in DB.props . The reason for that is:
https://github.com/radical/aspire/blob/af66351ffdc187cb707c242be1f30953d9e2d0d9/tests/Directory.Build.props#L18-L20

```xml
    <RunTestOnHelix Condition="'$(IsTestSupportProject)' != 'true' and
                                  ('$(IsHelixOnlyTestProject)' == 'true' or
                                    ('$(IsNonHelixOnlyTestProject)' != 'true' and '$(DefaultToRunningTestsOnHelix)' == 'true'))">true</RunTestOnHelix>
```